main.py

```diff
@@ -0,0 +1,12 @@
+from tic_tac_toe import *
+
+
+def main():
+    print(rules())
+    print(open_message())
```

Comment:
Don't use print. Study on how to use the logging module in python and use it instead